.github/pull_request_template.md

```diff
@@ -1,18 +1,33 @@
-*Note: Please remember to review the [contribution guidelines](https://github.com/DataDog/datadog-lambda-layer-rb/blob/master/CONTRIBUTING.md)
-if you have not yet done so.*
+<!--- Please remember to review the [contribution guidelines](https://github.com/DataDog/datadog-lambda-layer-python/blob/master/CONTRIBUTING.md) if you have not yet done so._  --->
```

Comment:
Should we copy the Python contribution guidelines here and link to them in the Ruby repo?

Reply:
That's a good catch.